packages/pwa-kit-create-app/package.json

```diff
@@ -23,11 +23,16 @@
     "lint": "eslint \"**/*.{js,jsx}\"",
     "lint:fix": "npm run lint -- --fix"
   },
+  "engines": {
+    "node": "^12.0.0 || ^14.0.0",
+    "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+  },
   "dependencies": {
     "commander": "^2.20.0",
     "deepmerge": "2.2.1",
     "fs-extra": "7.0.1",
     "inquirer": "7.0.5",
+    "semver": "^7.3.2",
```

Comment:
Moving this dependency as a production one. It used to be a dev dependency, but strangely were not being used.